src/content/en/updates/2018/12/badging-api.md

```diff
@@ -81,6 +81,14 @@ Examples of sites that may use this API include:
    running as an installed PWA (in your task bar or dock).
 3. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
+   
+Note: While the Badging API *in Chrome* requires an installed app
+with an icon that actually can be badged, we advise against
+making calls to the Badging API dependent on the install state.
```

Comment:
If it is advised against making calls to the Badging API based on the install state, based on what should the API be called?
If I understand this correctly, the intent here is to say that having an installed app does not guarantee that the Badging API will work, even when it's supported. Is this correct?

Reply:
Wait, after reading the title of this pull request again, I realized that this is not what you're trying to say. Instead, you're saying that the Badging API, when available, can be called even if the app is not installed. Is this correct?

Reply:
Yes. The Badge API should be called if the Badge API exists and you want to set a badge. Don't make assumptions about what circumstances the browser will show it. It is a "best effort" API.

Reply:
Thanks! I hope this feedback is dealt with in https://github.com/google/WebFundamentals/pull/7999/commits/9ec4212a74fbc2a285e9d162e1acada2852ed6e5.